sdk/digitaltwins/Microsoft.Azure.Management.DigitalTwins/src/Generated/IPrivateEndpointConnectionsOperations.cs

```diff
@@ -136,7 +137,7 @@ public partial interface IPrivateEndpointConnectionsOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<PrivateEndpointConnection>> CreateOrUpdateWithHttpMessagesAsync(string resourceGroupName, string resourceName, string privateEndpointConnectionName, PrivateEndpointConnectionProperties properties, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Breaking change due to type change

Reply:
Same here - not an intended change. I may rectify by bumping the major version.

Reply:
I've added PrivateEndpointConnectionProperties that inherits from ConnectionProperties. Hopefully that resolves the issue.